Resources/public/templates/ding2/slides/event/event.json

```diff
@@ -0,0 +1,79 @@
+{
+  "id": "event",
+  "type": "slide",
+  "name": "Arrangement",
+  "icon": "event.png",
+  "paths": {
+    "live": "event-edit.html",
+    "edit": "event-edit.html",
+    "preview": "event-preview.html",
+    "css": "event.css",
+    "js": "bundles/kkbding2integration/templates/js/eventSlide.js"
+  },
+  "slide_type": "event",
+  "media_type": "image",
+  "script_id": "event",
+  "orientation": "portrait",
+  "ideal_dimensions": {
+    "width": "1080",
+    "height": "1920"
+  },
+  "empty_options": {
+    "fontsize": 60,
+    "responsive_fontsize": 4,
+    "bgcolor": "#fff",
+    "headercolor": "#4e4a47",
```

Comment:
Would be nice to have this in an external configuration-file - but I'm cool with figuring out how to do that in a future slide.

Reply:
Wouldn't that be a core change needed?